verl/experimental/agent_loop/tool_agent_loop.py

```diff
@@ -31,14 +31,15 @@
 @register("tool_agent")
 class ToolAgentLoop(AgentLoopBase):
     @classmethod
-    def init_class(cls, config, tokenizer, **kwargs):
+    def init_class(cls, config, processing_class, **kwargs):
```

Comment:
Please a unit test for multimodal like this:
https://github.com/volcengine/verl/blob/main/tests/experimental/agent_loop/test_basic_agent_loop.py